Samples/Desktop/D3D12Raytracing/src/D3D12RaytracingSimpleLighting/readme.md

```diff
@@ -0,0 +1,47 @@
+# D3D12 Raytracing Simple Lighting sample
```

Comment:
Although you can include this in your change, you don't have to now because of PR #429. It might make your diff cleaner to not have to re-add it here.